src/webgl/material.js

```diff
@@ -211,29 +211,53 @@ p5.prototype.createShader = function(vertSrc, fragSrc) {
  * <div modernizr='webgl'>
  * <code>
  * function setup() {
- *   let frag = `precision mediump float;
- *   varying mediump vec2 vTexCoord;
+ *   let fragSrc = `precision highp float;
```

Comment:
Are we good to use `highp` as the default in examples? I guess it's only certain old mobile hardware that would care about mediump, and it's becoming less of a matter over time.

Reply:
Yep, no issues there!